### ISSUE TITLE
Crashes when acknowledging/dequeuing more than 1 job

### ISSUE THREAD

**Issue by ahkjeldsen** [ziyasal/Disque.Net#3]
Was trying to empty my queue as I was just testing some stuff.

```
var jobs = disque.GetJob(10, 1000, "test");
while (jobs.Count > 0)
{
	disque.Ackjob(jobs.Select(x => x.Id).ToArray());
	jobs = disque.GetJob(10, 1000, "test");
}
```

Throws:
> An unhandled exception of type 'CSRedis.RedisException' occurred in csredis.dll
Additional information: BADID Invalid Job ID format.

Also crashes when I substituted `1000` for `2`.

**Comment by aclemmensen** [comment]
Should be fixed by #7

**Comment by ahkjeldsen** [comment]
Fixed by https://github.com/ziyasal/Disque.Net/pull/7
